src/main/java/com/minecolonies/entity/ai/citizen/builder/EntityAIStructureBuilder.java

```diff
@@ -269,6 +269,26 @@ else if(colony != null)
         workOrder.setCleared(false);
     }
 
+    /**
+     * Gets a rotation from a block facing.
+     * @param facing the block facing.
+     * @return the int rotation.
+     */
+    private static int getRotationFromFacing(final EnumFacing facing)
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused private "getRotationFromFacing" method. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AUnusedPrivateMethod)